### DISCUSSION TITLE
Add your organization to our AirSim "Who's Using" list!

### DISCUSSION THREAD

**Discussion by jonyMarino** [microsoft/AirSim#3738] [General]
Hi,
Simplifying the process of being added to our list of [who is using airsim](https://microsoft.github.io/AirSim/who_is_using/), leave a comment here with the name of your organization and the link to its website.

**Comment by main2stels** [comment]
HI, @jonyMarino We use machine vision for ultralight trike. https://deltacnc.ru ClearSky LLC

**Comment by antoniosmakro** [comment]
Hi, @jonyMarino. Our company Myned AI https://www.myned.ai/ is using Airsim to train autonomous UAVs with reinforcement learning.

**Comment by meidachen** [comment]
Hi @jonyMarino , I'm a researcher at the University of Southern California Institute for Creative Technologies (ICT), our project named STPLS3D (www.stpls3d.com) used AirSim for rendering the RGB images along with the annotations and depth maps for creating photogrammetry point cloud database for semantic and instance segmentation.

**Reply by jonyMarino** [reply]
@meidachen that page is not available.

**Reply by meidachen** [reply]
Hi @jonyMarino, the site is currently offline, I will let you know once it is back.

Thanks.

**Reply by meidachen** [reply]
Hi @jonyMarino , I have made the website online again. Please take a look and add it to the list if possible. Thank you!

**Comment by Gastastrophe** [comment]
Hello @jonyMarino. I am a researcher at Central Michigan University that is part of a team that studies Flight simulation, including real time motion simulation and more recently autonomous piloting. I use Airsim for generating images of objects in Unreal (the camera vision mode) and for training and testing path planning algorithms using reinforcement learning and proportional navigation. Our ultimate goal is to train a quadcopter to fly through an obstacle course and have people race it from inside our motion simulator.

http://www.waynenterprises.com/research

**Comment by Erickrk** [comment]
Hello, I was a grad student for [this](http://www.cefet-rj.br/index.php) University here in Brazil and there we've developed a Virtual Reality Simulation of Autonomous Solar Plants Inspections with Unmanned Aerial Systems using AirSim.
Our work was published at IEEEXplore by AIRPHARO, which you can see [here](https://ieeexplore.ieee.org/document/9571060).

Please let me know if you need anything else!

**Reply by jonyMarino** [reply]
@Erickrk, excellent work you did with AirSim. We want to include ongoing projects that are using AirSim. Is that your case?

**Reply by Erickrk** [reply]
Hello @jonyMarino ! We've finished research after this publication. Some other papers came from the same scope but we're not an active group of our lab right now.
